response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_collection_instrument(short_name, period, file):
+    logger.debug('Retrieving collection exercise details', short_name=short_name, period=period)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/collection-instrument/{short_name}/{period}'
+    response = requests.post(url, files={"file": (file.filename, file.stream, file.mimetype)})
+    if response.status_code != 200:
```

Comment:
This request now returns `201` in backstage

Reply:
Good spot